### PR TITLE
Token Modal confirm for removal

### LINKED ISSUE #1272: Confirmation screen when trying to revoke PAT
### Is your suggested enhancement related to a problem? Please describe.

Yes, It's easy to missclick the "Revoke token" button since it's right next to the "Edit token" button.

### Describe the solution you'd like

A confirmation screen would appear when trying to revoke token.

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Mysterious-Dev**
All is in the title ^^.

Close #1272

### CODE REVIEW

**Comment by Mysterious-Dev** [comment]
Now, it's ready for review ^^

File: pages/settings/pats.vue [hunk 1]
```diff
@@ -190,6 +214,8 @@ const name = ref(null)
 const scopesVal = ref(0)
 const expires = ref(null)
 
+const deletPatIndex = ref(null)
```
Comment (triphora) [inline pages/settings/pats.vue:217]:
```suggestion
const deletePatIndex = ref(null)
```

File: pages/settings/pats.vue [hunk 2]
```diff
@@ -129,15 +136,32 @@
         >
           <EditIcon /> Edit token
         </button>
-        <button class="iconified-button raised-button" @click="removePat(pat.id)">
+        <button
+          class="iconified-button raised-button"
+          @click="
+            () => {
+              deletPatIndex = pat.id
```
Comment (triphora) [inline pages/settings/pats.vue:143]:
```suggestion
              deletePatIndex = pat.id
```

File: pages/settings/pats.vue [hunk 3]
```diff
@@ -1,5 +1,12 @@
 <template>
   <div class="universal-card">
+    <ModalConfirm
+      ref="modal_confirm"
+      title="Are you sure you want to delete this token?"
+      description="This will remove this token forever (like really forever)."
+      proceed-label="Delete this token"
+      @proceed="removePat(deletPatIndex)"
```
Comment (triphora) [inline pages/settings/pats.vue:8]:
```suggestion
      @proceed="removePat(deletePatIndex)"
```

**Review by triphora — CHANGES_REQUESTED** [review]
Looks great, just a spelling issue

**Comment by Mysterious-Dev** [comment]
> Looks great, just a spelling issue

Done ^^.

**Review by triphora — APPROVED** [review]

**Pull request merged**